src/main/java/org/broadinstitute/dsm/model/elastic/filter/splitter/DateLowerSplitter.java

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.dsm.model.elastic.filter.splitter;
```

Comment:
Looks like DateGreaterSplitter to me, could we not just use a DateSplitter and call it with the needed split parameter?

Reply:
DateGreaterSplitter extends GreaterThanEqualsSplitter and DateLowerSplitter extends LessThanEqualsSplitter, one general class won't be able to extend two different class, but to avoid duplication of the same code we will add helper class for it.